enclave-runtime/src/lib.rs

```diff
@@ -566,13 +566,18 @@ fn sync_parentchain_internal(blocks_to_sync: Vec<SignedBlock>) -> Result<()> {
 /// Triggers the import of parentchain blocks when using a queue to sync parentchain block import
 /// with sidechain block production.
 ///
-/// Imports all blocks in the queue except the latest one, which has to be triggered by the sidechain component.
+/// Imports all blocks in the current queue. This ensures the first validateer of a parentchain
+/// syncs up to the parentchain block where itself is registered. Otherwise the `authority_set` will
+/// return None, resulting in not importing any parentchain blocks and an endless loop (#589).
```

Comment:
I don't understand this sentence :(

Reply:
Removed in favor of the comments / explanations in `main.rs` This function may be called for different purposes, so the comment here is prone to be misleading anyway.